packages/manager/modules/vps/src/additional-disk/vps-additional-disk.html

```diff
@@ -51,18 +56,26 @@
                 </span>
             </oui-datagrid-column>
             <oui-action-menu data-compact data-placement="end">
-
                 <!--Upgrade disk: increase capacity-->
-                <oui-action-menu-item data-disabled="!$ctrl.isVpsNewRange || !$ctrl.upgradableDisks.length"
-                                      data-on-click="$ctrl.goToUpgradeDisk($row)">
-                    <span data-translate="vps_additional_disk_actions_increase_disk"></span>
+                <oui-action-menu-item
+                    data-disabled="!$ctrl.isVpsNewRange || !$ctrl.upgradableDisks.length"
+                    data-on-click="$ctrl.goToUpgradeDisk($row)"
+                >
+                    <span
+                        data-translate="vps_additional_disk_actions_increase_disk"
+                    ></span>
                 </oui-action-menu-item>
 
                 <!--Terminate a disk-->
-                <oui-action-menu-item data-on-click="$ctrl.goToTerminateDisk($row)">
-                    <span data-translate="vps_additional_disk_actions_terminate"></span>
+                <oui-action-menu-item
+                    data-on-click="$ctrl.goToTerminateDisk($row)"
+                    data-disabled="!$ctrl.canAdditionalDiskBeTerminated()"
+                    data-oui-tooltip="{{:: ($ctrl.canAdditionalDiskBeTerminated() ? '' : 'vps_additional_disk_actions_terminate_disabled_info') | translate}}"
```

Comment:
Would it be more simpler to just hide the `oui-action-menu-item` to avoid tooltip calculation?

Reply:
This is a particular case where VPS has a commitment. So Additional Disk cannot be terminated. That's the reason for disabling instead of hiding.